### PR TITLE
[BUG] Add tag `skip-inverse-transform` to ColumnSelect.

### LINKED ISSUE #5205: [BUG] ColumnSelect missing tag`"skip-inverse-transform"`
**Describe the bug**

The ColumnSelect is missing the tag  `"skip-inverse-transform"`. Thus, a TransformedTargetForecaster subsetting the `y` input will fail when calling predict.

**To Reproduce**

```python
from sktime.forecasting.naive import NaiveForecaster
from sktime.transformations.series.subset import ColumnSelect
from sktime.datasets import load_longley
y = load_longley()[1][["GNP", "UNEMP"]]
fc = ColumnSelect(["GNP"]) * NaiveForecaster()
fc.fit(y)
fc.predict(fh=[1])
```

**Expected behavior**

I would expect that the forecast is performed only on the selected time series without raising an error.

**Additional context**

This would be fixed by just adding the tag `skip-inverse-transform` to ColumnSelect.

### PR DESCRIPTION
**Pull request by benHeid**
#### Reference Issues/PRs
Fixes #5205 

#### What does this implement/fix? Explain your changes.
Add the missing tag `skip-inverse-transform` so that inverse_transform is skipped if ColumnSelect is part of a pipeline. 

#### Does your contribution introduce a new dependency? If yes, which one?
No

#### Did you add any tests for the change?
Yes

##### For all contributions
- [X] I've added myself to the [list of contributors](https://github.com/sktime/sktime/blob/main/CONTRIBUTORS.md) with any new badges I've earned :-)
  How to: add yourself to the [all-contributors file](https://github.com/sktime/sktime/blob/main/.all-contributorsrc) in the `sktime` root directory (not the `CONTRIBUTORS.md`). Common badges: `code` - fixing a bug, or adding code logic. `doc` - writing or improving documentation or docstrings. `bug` - reporting or diagnosing a bug (get this plus `code` if you also fixed the bug in the PR).`maintenance` - CI, test framework, release.
  See here for [full badge reference](https://allcontributors.org/docs/en/emoji-key)
- [ ] Optionally, I've added myself and possibly others to the [CODEOWNERS](https://github.com/sktime/sktime/blob/main/CODEOWNERS) file - do this if you want to become the owner or maintainer of an estimator you added.
  See here for further details on the [algorithm maintainer role](https://www.sktime.net/en/latest/get_involved/governance.html#algorithm-maintainers).
- [x] The PR title starts with either [ENH], [MNT], [DOC], or [BUG]. [BUG] - bugfix, [MNT] - CI, test framework, [ENH] - adding or improving code, [DOC] - writing or improving documentation or docstrings.

##### For new estimators
- [ ] I've added the estimator to the API reference - in `docs/source/api_reference/taskname.rst`, follow the pattern.
- [ ] I've added one or more illustrative usage examples to the docstring, in a pydocstyle compliant `Examples` section.
- [ ] If the estimator relies on a soft dependency, I've set the `python_dependencies` tag and ensured
  dependency isolation, see the [estimator dependencies guide](https://www.sktime.net/en/latest/developer_guide/dependencies.html#adding-a-soft-dependency).

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -1740,6 +1740,7 @@
       "name": "Benedikt Heidrich",
       "profile": "https://github.com/benheid",
       "contributions": [
+        "bug",
```
Comment (fkiraly) [inline .all-contributorsrc:1743]: you should have more here, now that I see.

design, ideas -> pipelines etc
doc -> writing or fixing docstrings
example -> in the docstrings and tutorial notebooks
mentoring -> obviously, what it says
question -> on discord
review -> many PRs
tutorial -> the pydatas

Comment (fkiraly) [reply .all-contributorsrc:1743]: maybe separate PR to also have the reason in the commit message?

**Review by fkiraly — APPROVED** [review]
Makes sense, as discussed.

Would recommend to make a contributions update properly in another PR, looks like you haven't been tracking that properly.

File: .all-contributorsrc (hunk 1 above)
Comment (benHeid) [reply .all-contributorsrc:1743]: Yeah, I forgot it. I planned to do this successively, but I will create a new PR where I add the missing ones.

**Pull request merged**